modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -18,20 +22,29 @@ ldd --version
 
 ## Setting up the environment
```

Comment:
This heading is getting truncated in the navigation table. Would it be possible to shorten this to "Setup Environment"?

Reply:
Yeah, that sounds great!